### ISSUE TITLE
Hello from xplr

### ISSUE THREAD

**Issue by sayanarijit** [marcusbuffett/pipe-rename#34]
Hi, nice tool. I have migrated the [bach rename xplr hack](https://github.com/sayanarijit/xplr/wiki/Hacks#batch-rename) from [mmv](https://github.com/itchyny/mmv) to `pipe-rename` because it supports reading filenames from stdin.

This solves the following problems:

- I don't have to deal with escaping file paths that include white spaces. I can just use `cat $XPLR_PIPE_RESULT_OUT | renamer`.
- The input is not limited to the maximum size of command-line arguments supported by the system.

Really neat thinking :+1:

Also, I like the custom rename command argument. That gave me this project idea of a more generic tool, let's call it `batch`.

So when I execute `ls | batch 'mv {} {}'`, editor will open

```
mv "a" "a"
mv "b" "b"
```

or `rm -rf {}`

```
rm -rf "a"
rm -rf "b"
```

Basically `xargs` but lets you edit and review the operation before executing, with a nice confirmation prompt.

**Comment by marcusbuffett** [comment]
Sweet! Happy to hear this was useful for you :D 

Love the `batch` idea, cool that it would be a two-for-one, in that it's doing a dry-run and fixing any issues, in one go.
